src/main/resources/content/cft/AWS-S3-buckets-are-accessible-to-public.json

```diff
@@ -1 +1,8 @@
-{"severity":"high","rule":"($.Resources.*[?(@.Type=='AWS::S3::Bucket')].Properties.AccessControl any equal PublicRead or $.Resources.*[?(@.Type=='AWS::S3::Bucket')].Properties.AccessControl any equal PublicReadWrite) and $.Resources.*[?(@.Type=='AWS::S3::Bucket')].Properties.WebsiteConfiguration any null","id":"bbb01285-7fc6-4649-85c0-6ab9f08bde4f","enabled":true,"resourceType":"s3","policy":"AWS S3 buckets are accessible to public"}
\ No newline at end of file
+{
+  "severity": "high",
+  "rule": "($.Resources.*[?(@.Type=='AWS::S3::Bucket')].Properties.AccessControl any equal PublicRead or $.Resources.*[?(@.Type=='AWS::S3::Bucket')].Properties.AccessControl any equal PublicReadWrite) and $.Resources.*[?(@.Type=='AWS::S3::Bucket')].Properties.WebsiteConfiguration any null",
```

Comment:
I still see the Website configuration check. It's just reformatting of the old rule. Looks like your changes were not updated.
$.Resources.*[?(@.Type=='AWS::S3::Bucket')].Properties.WebsiteConfiguration any null